Documentation/using-rkt-with-systemd.md

```diff
@@ -77,6 +77,32 @@ If the output contains `-XZ`, journal entries will not be available.
 
 ## Managing pods as systemd services
 
+### Notifications
+
+rkt can notify systemd on the host that all apps of the pod are ready using the [sd_notify][sd_notify] mechanism. To make use of this feature, you need to set the annotation `appc.io/executor/supports-systemd-notify` to true in the image manifest whenever the app supports sd\_notify.
+```
+{
+	"acKind": "ImageManifest",
+	"acVersion": "0.8.4",
+	"name": "coreos.com/etcd",
+	...
+	"app": {
+		"exec": [
+			"/etcd"
+		],
+		...
+	},
+	"annotations": [
+	    "name": "appc.io/executor/supports-systemd-notify",
+	    "value": "true"
+	]
+}
+```
+This feature is always available when using the "coreos" stage1 flavor. If you use the "host" stage1 flavor (e.g. using Fedora RPM or Debian deb package), you will also need systemd >= v231.
+Run in a terminal window this command `sudo systemd-run --unit=test --service-type=notify rkt run --insecure-options=image /path/to/your/app/image` then check the status periodically with `sudo systemctl status test`.
```

Comment:
"Execute `sudo systemd-run ...`, and check the status using `systemctl status test`"